wiki/markdown/artifact-created-event-aggregation.md

```diff
@@ -100,7 +100,7 @@ is configured in the application.properties file as “waitlist.collection.name
 The event is stored with the aim that if an aggregated object is created later
 with links to this specific event then it can be merged with that object. The
 event storage life time is set in the properties file as
```

Comment:
Instead of duplicating the information about properties and their names maybe we should just link from here to the configuration.md to avoid this hassle in the future? 😄

Reply:
Something like this instead? 'You can read about the configuration regarding the waitlist [here](configuration.md#waitlist)'
There are probably more places that could be linked though, but perhaps those can be moved into another issue?

Reply:
I'll change the doc in the places you've spotted.